src/Documentation/Deployment-and-Operations/Configuration-Guide/Typical-Configurations.md

```diff
@@ -126,7 +126,7 @@ var gateways = new IPEndPoint[]
     new IPEndPoint(OTHER_SILO__IP_ADDRESS_N, 11111),
 };
 var client = new ClientBuilder()
-    .UseStaticClustering(gateways)
+    .UseStaticClustering(gatewayPort: 30000)
```

Comment:
I don't think this compiles. UseStaticClusterting take a `IPEndPoint[] endpoints` as param, not just port. I think what @ReubenBond means in the issue is the port in `gateways` are supposed to be 30000, instead of 11111